AppDB/groomer.py

```diff
@@ -177,22 +193,42 @@ def remove_deprecated_dashboard_data(self, model_type):
 
   def remove_old_dashboard_data(self):
     """ Removes old statistics from the AppScale dashboard application. """
+    last_cursor = None
+    last_model = None
+    if len(self.groomer_state) > 1 and self.groomer_state[0] == 'dashboard':
```

Comment:
Can you add a comment here explaining that >1?

Also, 1 and "dashboard" should be constants defined above.